### DISCUSSION TITLE
Kibana "Application Not Found"

### DISCUSSION THREAD

**Discussion by trb143** [Security-Onion-Solutions/securityonion#9672] [Unsupported Versions]
Having just installed a fresh version of 2.3.200 I am seeing data in Kibana but when I select to drill down to finer details I get the following.

![image](https://user-images.githubusercontent.com/1388819/215470767-18f03436-2cc6-4904-b52a-988306f9a21f.png)

There is data 

![image](https://user-images.githubusercontent.com/1388819/215471009-e352eb99-bd99-4f44-850a-241925a6b277.png)

The problem happens for all the modules like zeek etc.

How can this be fixed?

**Comment by InfosecGoon** [comment]
Unfortunately, this is an issue with the version of Elastic in .200 that is fixed in Elastic 8.6. It's being tracked in this issue: https://github.com/Security-Onion-Solutions/securityonion/issues/9594
